doc/releases/release-notes-2.1.rst

```diff
@@ -141,4 +660,305 @@ release:
+* :github:`19820` - Bluetooth: Host: Unable to use whitelist in peripheral only build
+* :github:`19818` - Compiler error for counter example (nRF52_pca10040)
+* :github:`19811` - native_posix stack smashing
+* :github:`19802` - Zephyr was unable to find the toolchain after update to zephyr version 1.13.0
+* :github:`19795` - bt_gatt_attr_next returns first attribute in table for attributes with static storage.
+* :github:`19791` - How to use CMSIS DSP Library on nRF52832 running zephyr LTS Version(V1.14) ?
+* :github:`19783` - floating point in C++ on x86_64 uses SSE
+* :github:`19775` - net_calc_chksum: Use of un-initialized memory on 64 bit targets
+* :github:`19769` - CONFIG_FLASH_SIZE should be CONFIG_FLASH_END and specified in hex
+* :github:`19767` - Bluetooth: Mesh: Provision Random buffer has too small size
+* :github:`19762` - tests/net/lib/tls_credentials failed on sam_e70_xplained board.
+* :github:`19759` - z_arch_switch() passed pointer to NULL outgoing switch handle on dummy thread context switch
+* :github:`19748` - k_sleep(K_FOREVER) behavior unexpected
+* :github:`19734` - "make gdbserver" doesn't work properly for qemu_x86_long
+* :github:`19724` - Bluetooth: Mesh: Receiving an access message
+* :github:`19722` - Settings: settings_file_save_priv() use of uninitialized variable
+* :github:`19721` - samples/bluetooth/ipsp does not respond to pings from Linux
+* :github:`19717` - Add provisions for supporting multiple CMSIS variants
+* :github:`19701` - mem_pool_threadsafe sporadic failures impacting CI
+* :github:`19697` - tests/subsys/fs/fat_fs_api uses unitialized variables
+* :github:`19692` - [TOPIC-GPIO] gpi_api_1pin test failures
+* :github:`19683` - nrf: clock reimplementation breaks test
+* :github:`19678` - Noticeable delay between processing multiple client connection requests (200ms+)
+* :github:`19660` - missing file reference in samples/sensor/ti_hdc doc
+* :github:`19649` - [TOPIC-GPIO]: Replace GPIO_INT_DEBOUNCE with GPIO_DEBOUNCE
+* :github:`19638` - Bluetooth: Mesh: Provisioning Over PB-ADV
+* :github:`19629` - tinycbor buffer overflow causing mcumgr image upload failure
+* :github:`19612` - ICMPv6 packet is routed to wrong interface when peer is not found in neighbor cache
+* :github:`19604` - Bluetooth: ATT does not release all buffers on disconnect
+* :github:`19603` - addition to winbond,w25q16.yaml required for SPI CS to be controlled by driver
+* :github:`19592` - Request new repository to host the Eclipse plugin for building Zephyr applications
+* :github:`19569` - nRF RTC Counter with compile time decision about support of custom top value
+* :github:`19560` - Console on CDC USB  crashes when CONFIG_USB_COMPOSITE_DEVICE=y
+* :github:`19552` - [TOPIC-GPIO]: Support for legacy interrupt configuration breaks new API contract
+* :github:`19550` - drivers/pcie: `pcie_get_mbar()` should return a `void *` not `u32_t`
```

Comment:
The script that creates these lists of issues needs to scan for meaningful reST syntax and escape those so they're not interpreted by Sphinx.  In this case replace `` ` `` with `` ' `` (back-tick with apostrophe)